src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java

```diff
@@ -357,7 +351,7 @@ public void onResponse( @NotNull Call call, @NotNull Response response )
                         scope.close();
                         span.end();
 
-                        logger.trace( "Success: " + call.request().url() + " -> " + response.code() );
+                        logger.info( "Success: [" + method + "] " + url + " -> " + response.code() );
```

Comment:
I think we should avoid to use "+" for string in logging, instead use parameterized log( use {} ).

Reply:
Updated.